scalding-core/src/main/scala/com/twitter/scalding/graph/Vertex.scala

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2012 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package com.twitter.scalding.graph
+
+case class Vertex[T, S](id: T, attr: S)(implicit ord: Ordering[T]) extends Ordered[Vertex[T, _]] {
```

Comment:
should not have an implicit `Ordering`.

You should do:
```scala
object Vertex {
  implicit def ordering[T: Ordering, S: Ordering]: Ordering[Vertex[T, S]] = ...
}
```
I am not a fan of saying two things are equal in an ordering, but they have different attributes. Do we really need that?